docs/wiki/installation/cli-flags.md

```diff
@@ -330,6 +330,20 @@ Comma-delimited list of table names to be disabled. This allows osquery to be la
 
 Maximum file read size. The daemon or shell will first 'stat' each file before reading. If the reported size is greater than `read_max` a "file too large" error will be returned.
 
+## Linux-only runtime control flags
+
+`--malloc_trim_threshold=200`
+
+Memory threshold in MB used to decide when a malloc_trim will be called to reduce the retained memory.
+When the flag is not provided, the value will be chosen automatically between 80% of the `watchdog_memory_limit` if the watchdog is not disabled and 200MB in the case it is.
```

Comment:
How do you think I should rephrase that?
Making the "positive" example: when the flag is provided, the threshold will be the one provided, otherwise the default is automatic.
The automatic behavior changes on the watchdog being enabled or not; if enabled, then the value will be 80% of the `watchdog_memory_limit`, otherwise it will be 200MB.

Reply:
Maybe something like:
```
The default behavior depends on the watchdog. When it is enabled, the default 80% of `watchdog_memory_limit`, otherwise it defaults to 200MB.
```

I'd take it as is though -- I did approve it.